src/nurse/tests/test_views.py

```diff
@@ -784,3 +784,55 @@ def test_post_only_staff(self, user, client):
         assert response.json() == {
             "detail": "You do not have permission to perform this action."
         }
+
+
+@pytest.fixture
+def user_profile(user):
+    return UserOneSignalProfile.objects.create(user=user, subscription_id="123456789")
```

Comment:
Let's call it `one_signal_profile` so we don't confuse it with the other user profile that we have (e.g. the ProfileView)

Reply:
I updated the name